test/unit/lib/plugins/package/lib/packageService.test.js

```diff
@@ -482,82 +106,90 @@ describe('lib/plugins/package/lib/packageService.test.js', () => {
       });
     });
 
-    it.skip('TODO: should exclude default plugins localPath', () => {
+    it('should exclude default plugins localPath', () => {
       // Confirm ".serverless-plugins/index.js" is not packaged
-      //
+      expect(fnIndividualZippedFiles).to.not.include('.serverless-plugins/index.js');
       // Replaces
       // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L87-L96
     });
 
-    it.skip('TODO: should support `package.exclude`', () => {
+    it('should support `package.exclude`', () => {
       // Confirm "dir1/subdir1/index.js" is not packaged
+      expect(fnIndividualZippedFiles).to.not.include('dir1/subdir1/index.js');
       // Confirm "dir1/subdir3/index.js" is packaged
-      //
+      expect(fnIndividualZippedFiles).to.include('dir1/subdir3/index.js');
       // Replace
       // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L128-L145
       // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L637-L649
     });
 
-    it.skip('TODO: should support `package.include`', () => {
+    it('should support `package.include`', () => {
       // Confirm "dir1/subdir2/index.js" is packaged
+      expect(fnIndividualZippedFiles).to.include('dir1/subdir2/index.js');
       // Confirm "dir1/subdir2/subsubdir1/index.js" is not packaged
+      expect(fnIndividualZippedFiles).to.not.include('dir1/subdir2/subsubdir1/index.js');
       // Confirm "dir1/subdir2/subsubdir2/index.js" is packaged
-      //
+      expect(fnIndividualZippedFiles).to.include('dir1/subdir2/subsubdir2/index.js');
       // Replaces
       // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L43-L50
       // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L625-L635
       // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L651-L662
     });
 
-    it.skip('TODO: should support `functions[].package.individually`', () => {
+    it('TODO: should support `functions[].package.individually`', () => {
       // Confirm there's functions.fnIndividual.package.artifact
+      // Not sure if valid. To be checked.
+      expect(fnIndividualZippedFiles).to.include('artifact.zip');
```

Comment:
I believe in this case it would make sense to validate that artifacts for individually packaged functions exist

Reply:
What is your opinion on this one?

Reply:
I agree this test is needed. I just wasn't sure my assertion was correct, hence the comment.

Reply:
This assertion should be a bit different - right now we're checking if the individually packaged artifact includes `artifact.zip` which doesn't tell us much. Let's instead check that `'fnIndividual.zip'` was set as `package.artifact` for that function (as suggested in the comment in code)

Reply:
Should be fixed with 752d30e4b469bb227092c037e44c3e653d70270b.